tox.ini

```diff
@@ -12,3 +12,6 @@ install_command = pip install --pre {opts} {packages}
 commands = flake8 sops
 deps =
     flake8
+
+[flake8]
+ignore = E722,E226
```

Comment:
I don't think globally ignoring these lint errors is a good idea. Can we either fix the causes of this lint errors or #noqa the appropriate lines?

Reply:
I’ll take a look at the E722s. E226 is off by default, but once you specify a custom ignore, all the off by default warnings are no longer ignored. 😞

Reply:
fixed ✔️

Reply:
nope, they're gone..